apps/HelloAndroidCamera2/README.md

```diff
@@ -0,0 +1,80 @@
+HelloAndroidCamera2 is a simple application which uses Halide to
+process images streamed from the Android camera2 API. It reads every
+frame into the CPU via an ImageReader and uses Halide to either blit
+the frame to the output surface (converting between YUV formats), or
+apply an edge detector on the luma channel. This example requires a
+phone or tablet that supports the camera2 API (level 21 or above).
+
+This application builds for multiple native ABIs. (At present armeabi,
+armeabi-v7a, arm64-v8a, mips, x86_64, and x86 are supported. mips64 is
+not presently working.) Halide code is generated for each architecture.
+
+This build is meant to use Android command line tools. (And IDE is not
+required.) In order to build, the following will be required:
+
+*Android NDK -- This can be download here:
```

Comment:
Thanks so much for including details on what Android tools are needed and some details on configuration!